+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Tue Sep  5 09:48:50 2023 - INFO  - Start to prepare the test environment.
+ P_SSH_CMD --node 2 --cmd 'which sshpass'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'which sshpass'
which: no sshpass in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin)
+ [[ x1 != \x\0 ]]
+ P_SSH_CMD --node 2 --cmd 'yum install sshpass -y'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'yum install sshpass -y'
Errors during downloading metadata for repository 'mainline':
  - Curl error (28): Timeout was reached for https://mirror.iscas.ac.cn/openeuler-sig-riscv/openEuler-RISC-V/testing/20230830/v0.2/repo/23.09/mainline/repodata/repomd.xml [Failed to connect to mirror.iscas.ac.cn port 443 after 30000 ms: Timeout was reached]
  - Curl error (28): Timeout was reached for https://mirror.iscas.ac.cn/openeuler-sig-riscv/openEuler-RISC-V/testing/20230830/v0.2/repo/23.09/mainline/repodata/repomd.xml [Failed to connect to mirror.iscas.ac.cn port 443 after 30001 ms: Timeout was reached]
  - Curl error (28): Timeout was reached for https://mirror.iscas.ac.cn/openeuler-sig-riscv/openEuler-RISC-V/testing/20230830/v0.2/repo/23.09/mainline/repodata/repomd.xml [Failed to connect to mirror.iscas.ac.cn port 443 after 30003 ms: Timeout was reached]
Error: Failed to download metadata for repo 'mainline': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
+ sshpass_status=0
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Tue Sep  5 09:51:13 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Tue Sep  5 09:51:14 2023 - INFO  - Start to run test.
++ firewall-cmd --state
oe_test_firewalld_zone_migration.sh: line 33: firewall-cmd: command not found
+ CHECK_RESULT '' running 0 'firewalld.service not running'
+ actual_result=
+ expect_result=running
+ mode=0
+ error_log='firewalld.service not running'
+ exit_mode=0
+ '[' -z '' ']'
+ LOG_ERROR 'Missing actual error code.'
+ message='Missing actual error code.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Missing actual error code.'
Tue Sep  5 09:51:14 2023 - ERROR - Missing actual error code.
+ return 1
+ firewall-cmd --delete-zone=zonetest --permanent
oe_test_firewalld_zone_migration.sh: line 34: firewall-cmd: command not found
+ firewall-cmd --reload
oe_test_firewalld_zone_migration.sh: line 35: firewall-cmd: command not found
+ firewall-cmd --get-zones
oe_test_firewalld_zone_migration.sh: line 36: firewall-cmd: command not found
+ grep zonetest
+ CHECK_RESULT 1 0 1 'default firewall zone check fail'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='default firewall zone check fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ CMD_String='sshpass -p openEuler12#$ ssh -o '\''StrictHostKeyChecking no'\'' root@10.0.0.2 '\''cat /etc/os-release'\'''
+ P_SSH_CMD --node 2 --cmd 'sshpass -p openEuler12#$ ssh -o '\''StrictHostKeyChecking no'\'' root@10.0.0.2 '\''cat /etc/os-release'\'''
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'sshpass -p openEuler12#$ ssh -o '\''StrictHostKeyChecking no'\'' root@10.0.0.2 '\''cat /etc/os-release'\'''
bash: line 1: sshpass: command not found
+ CHECK_RESULT 127 0 0 'ssh NODE1_IPV4 fail'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ssh NODE1_IPV4 fail'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ssh NODE1_IPV4 fail'
+ LOG_ERROR 'ssh NODE1_IPV4 fail'
+ message='ssh NODE1_IPV4 fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ssh NODE1_IPV4 fail'
Tue Sep  5 09:51:21 2023 - ERROR - ssh NODE1_IPV4 fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_firewalld_zone_migration.sh line 40'
+ message='oe_test_firewalld_zone_migration.sh line 40'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_firewalld_zone_migration.sh line 40'
Tue Sep  5 09:51:22 2023 - ERROR - oe_test_firewalld_zone_migration.sh line 40
+ '[' 0 -eq 1 ']'
+ return 0
+ firewall-cmd --new-zone=zonetest --permanent
oe_test_firewalld_zone_migration.sh: line 42: firewall-cmd: command not found
+ firewall-cmd --reload
oe_test_firewalld_zone_migration.sh: line 43: firewall-cmd: command not found
+ firewall-cmd --get-zones
oe_test_firewalld_zone_migration.sh: line 44: firewall-cmd: command not found
+ grep zonetest
+ CHECK_RESULT 1 0 0 'add firewall zone check fail'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='add firewall zone check fail'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'add firewall zone check fail'
+ LOG_ERROR 'add firewall zone check fail'
+ message='add firewall zone check fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'add firewall zone check fail'
Tue Sep  5 09:51:22 2023 - ERROR - add firewall zone check fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_firewalld_zone_migration.sh line 45'
+ message='oe_test_firewalld_zone_migration.sh line 45'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_firewalld_zone_migration.sh line 45'
Tue Sep  5 09:51:23 2023 - ERROR - oe_test_firewalld_zone_migration.sh line 45
+ '[' 0 -eq 1 ']'
+ return 0
+ firewall-cmd --zone=zonetest --add-source=10.0.0.6
oe_test_firewalld_zone_migration.sh: line 47: firewall-cmd: command not found
+ firewall-cmd --zone=zonetest --add-port=65535/tcp
oe_test_firewalld_zone_migration.sh: line 48: firewall-cmd: command not found
++ firewall-cmd --list-all --zone=zonetest
oe_test_firewalld_zone_migration.sh: line 49: firewall-cmd: command not found
++ grep sources
++ awk '{print $NF}'
+ CHECK_RESULT '' 10.0.0.6 0 'add source fail for zonetest'
+ actual_result=
+ expect_result=10.0.0.6
+ mode=0
+ error_log='add source fail for zonetest'
+ exit_mode=0
+ '[' -z '' ']'
+ LOG_ERROR 'Missing actual error code.'
+ message='Missing actual error code.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Missing actual error code.'
Tue Sep  5 09:51:24 2023 - ERROR - Missing actual error code.
+ return 1
++ firewall-cmd --list-all --zone=zonetest
++ grep '^  ports'
oe_test_firewalld_zone_migration.sh: line 50: firewall-cmd: command not found
++ awk '{print $NF}'
+ CHECK_RESULT '' 65535/tcp 0 'add ports fail for zonetest'
+ actual_result=
+ expect_result=65535/tcp
+ mode=0
+ error_log='add ports fail for zonetest'
+ exit_mode=0
+ '[' -z '' ']'
+ LOG_ERROR 'Missing actual error code.'
+ message='Missing actual error code.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Missing actual error code.'
Tue Sep  5 09:51:25 2023 - ERROR - Missing actual error code.
+ return 1
+ P_SSH_CMD --node 2 --cmd 'sshpass -p openEuler12#$ ssh -o '\''StrictHostKeyChecking no'\'' root@10.0.0.2 '\''cat /etc/os-release'\'''
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'sshpass -p openEuler12#$ ssh -o '\''StrictHostKeyChecking no'\'' root@10.0.0.2 '\''cat /etc/os-release'\'''
Tue Sep  5 09:51:58 2023 - ERROR - Failed to connect the remote machine:10.0.0.6.
Tue Sep  5 09:51:58 2023 - ERROR - Authentication timeout.

+ CHECK_RESULT 7 0 1 'ssh NODE1_IPV4 fail. Firewalld zone migration is not supported'
+ actual_result=7
+ expect_result=0
+ mode=1
+ error_log='ssh NODE1_IPV4 fail. Firewalld zone migration is not supported'
+ exit_mode=0
+ '[' -z 7 ']'
+ '[' 1 -eq 0 ']'
+ test 7x == 0x
+ return 0
+ firewall-cmd --delete-zone=zonetest --permanent
oe_test_firewalld_zone_migration.sh: line 54: firewall-cmd: command not found
+ firewall-cmd --reload
oe_test_firewalld_zone_migration.sh: line 55: firewall-cmd: command not found
+ P_SSH_CMD --node 2 --cmd 'sshpass -p openEuler12#$ ssh -o '\''StrictHostKeyChecking no'\'' root@10.0.0.2 '\''cat /etc/os-release'\'''
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'sshpass -p openEuler12#$ ssh -o '\''StrictHostKeyChecking no'\'' root@10.0.0.2 '\''cat /etc/os-release'\'''
bash: line 1: sshpass: command not found
+ CHECK_RESULT 127 0 0 'ssh NODE1_IPV4 fail'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ssh NODE1_IPV4 fail'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ssh NODE1_IPV4 fail'
+ LOG_ERROR 'ssh NODE1_IPV4 fail'
+ message='ssh NODE1_IPV4 fail'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ssh NODE1_IPV4 fail'
Tue Sep  5 09:52:08 2023 - ERROR - ssh NODE1_IPV4 fail
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_firewalld_zone_migration.sh line 57'
+ message='oe_test_firewalld_zone_migration.sh line 57'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_firewalld_zone_migration.sh line 57'
Tue Sep  5 09:52:09 2023 - ERROR - oe_test_firewalld_zone_migration.sh line 57
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Tue Sep  5 09:52:10 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 3
+ test 3 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Tue Sep  5 09:52:10 2023 - ERROR - failed to execute the case.
+ exit 3
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Tue Sep  5 09:52:11 2023 - INFO  - Start to restore the test environment.
+ [[ x0 == \x\0 ]]
+ P_SSH_CMD --node 2 --cmd 'yum remove sshpass -y'
+ python3 /root/mugen/libs/locallibs/ssh_cmd.py --node 2 --cmd 'yum remove sshpass -y'
Exception (client): Error reading SSH protocol banner
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/paramiko/transport.py", line 2292, in _check_banner
    buf = self.packetizer.readline(timeout)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/paramiko/packet.py", line 374, in readline
    buf += self._read_timeout(timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/paramiko/packet.py", line 611, in _read_timeout
    raise socket.timeout()
TimeoutError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/paramiko/transport.py", line 2113, in run
    self._check_banner()
  File "/usr/lib/python3.11/site-packages/paramiko/transport.py", line 2296, in _check_banner
    raise SSHException(
paramiko.ssh_exception.SSHException: Error reading SSH protocol banner

Tue Sep  5 09:52:29 2023 - ERROR - Failed to connect the remote machine:10.0.0.6.
Tue Sep  5 09:52:29 2023 - ERROR - Error reading SSH protocol banner

+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Tue Sep  5 09:52:30 2023 - INFO  - End to restore the test environment.
